source/agora/test/Script.d

```diff
@@ -254,8 +254,10 @@ unittest
                 }))
         .array();
 
-    true_a_txs.each!(tx => node_1.putTransaction(tx));
-    true_b_txs.each!(tx => node_1.putTransaction(tx));
+    // We don't want to rely on gossip before we set time for nominate or we will not know
+    //  the chosen tx set for sure. We just send to all so that all will be included next block.
+    true_a_txs.each!(tx => nodes.each!(node => node.putTransaction(tx)));
+    true_b_txs.each!(tx => nodes.each!(node => node.putTransaction(tx)));
     network.expectHeightAndPreImg(Height(3), network.blocks[0].header);
```

Comment:
A few lines later the `true_a_txs` are sorted and compared to `txs` in `block`.

Reply:
This was ok before because we waited for the expected number of txs in this test. I think it should be ok now that they all receive them. Here we are not testing gossiping but script logic so I think it is ok.